src/ml/neural_net/style_transfer/mps_style_transfer_decoding_node.m

```diff
@@ -0,0 +1,69 @@
+#import <ml/neural_net/style_transfer/mps_style_transfer_decoding_node.h>
+#import <ml/neural_net/mps_layer_helper.h>
+
+@interface TCMPSStyleTransferDecodingNode () {
+  MPSCNNUpsamplingNearestNode *upsample;
```

Comment:
Instance variable names should begin with an underscore. Or better yet, define these as properties (with names without underscores), and that will auto-generate instance variables with the underscore prefix

Reply:
Can we have private properties?

Reply:
Corrected!